cli/template/nextjs/src/components/AppShell/internal/HeaderNavLink.tsx

```diff
@@ -10,7 +10,7 @@ export default function HeaderNavLink(route: ProofKitRoute) {
   const pathname = usePathname();
 
   if (route.type === "function") {
-    return <a className={classes.link}>{route.label}</a>;
+    return <div className={classes.link}>{route.label} onClick={route.onClick}</div>;
```

Comment:
for accessibility, this should be a button


```suggestion
    return <button className={classes.link} onClick={route.onClick}>{route.label}</button>;
```